woo-razorpay.php

```diff
@@ -588,13 +588,18 @@ private function getRedirectUrl($orderId)
         protected function getRazorpayPaymentParams($orderId)
         {
             $getWebhookFlag =  get_option('webhook_enable_flag');
-
+            $time = time();
            if (!empty($getWebhookFlag))
            {
                 if ($getWebhookFlag + 86400 < time())
```

Comment:
@ramth05 Is this condition correct? Are we verifying within 24 ours or after 24 hours? cc// @jampulanaveen

Reply:
@ChetanGN , yes it's correct. If we don't have time value when last webhook enabled/updated. We need to update it with the transaction. 
if it's already set then verify the 24 hours timeframe and verify the webhook again.